Documentation/index.html

```diff
@@ -44,15 +44,63 @@
         <section>
           <section class="section">
             
-            <h1 id='researchsuite' class='heading'>SageResearch</h1>
-<h3 id='authors' class='heading'>Authors</h3>
-
-<p>Sage Bionetworks</p>
-
+            <h1 id='sageresearch' class='heading'>SageResearch</h1>
+            
+            <p><a href="http://sagebionetworks.org">Sage Bionetworks</a> has spent four years developing mobile health study apps and played a role in supporting over 20 projects, using Research Kit as the basis for all of our iPhone development. Based on that experience, we have begun to generate a next-generation framework and are looking for the involvement of the broader Research Kit community to provide input on design goals and approaches, and contribute to development. This framework:</p>
+            
+            <ul>
+                <li>Keeps what is good and useful about ResearchKit, provides for a smooth transition from it, and allows ResearchKit 1.x components to be used as modules in Research apps;</li>
```

Comment:
```suggestion
                <li>Keeps what is good and useful about ResearchKit, provides for a smooth transition from it, and allows ResearchKit 2.x components to be used as modules in Research apps;</li>
```

Reply:
Yeah, I thought about that but I haven't really tested it against 2.x so ?